test/integration/model/optimistic_locking.test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const Support = require(__dirname + '/../support')
+  , DataTypes = require(__dirname + '/../../../lib/data-types')
+  , chai = require('chai')
+  , expect = chai.expect;
```

Comment:
The tests have not been refactored to ES6 yet, but the files who are refactored use one `const` statement per declaration